cloud/services/compute/networks/reconcile.go

```diff
@@ -130,6 +141,11 @@ func (s *Service) createOrGetRouter(ctx context.Context, network *compute.Networ
 			return nil, err
 		}
 
+		if s.scope.IsSharedVpc() {
+			log.Error(err, "Shared VPC is enabled, but could not find find existing router", "name", routerKey)
```

Comment:
Nit: "find" occurs twice in this error message.

Reply:
see similar comment.